caikit/runtime/grpc_server.py

```diff
@@ -103,6 +114,31 @@ def __init__(
                 self.inference_service.service, self.server
             )
 
+        if self.enable_inference_jobs:
```

Comment:
NIT: Should this be `enable_prediction_jobs`?

Reply:
I guess we say `enable_inference`, so it's probably fine as-is

Reply:
yeah that was a big pain point for me. I think enable_prediction_jobs is the correct answer but since we already use enable_inference I thought we should retain consistency

Reply:
Yeah, ug, sorry for the inconsistency!